sources/TileDB.CSharp/Interop/tiledb_walk_order_t.cs

```diff
@@ -1,7 +1,11 @@
 // <auto-generated/>
 
+using System;
+using TileDB.CSharp;
+
 namespace TileDB.Interop
 {
+    [Obsolete(Obsoletions.TileDBInteropMessage + " Use the TileDB.CSharp.WalkOrder type instead.", DiagnosticId = Obsoletions.TileDBInteropDiagId, UrlFormat = Obsoletions.SharedUrlFormat)]
```

Comment:
Still working on relocating some of these to `InteropAugments`? I regenerated using this branch and these were removed by `ClangSharpPInvokeGenerator`.